spec/features/new_device_tracking_spec.rb

```diff
@@ -63,7 +63,7 @@
       expect(UserMailer).to have_received(:new_device_sign_in).
         with(
           user.email_addresses.first,
-          device.last_used_at.strftime('%B %-d, %Y %H:%M'),
+          device.last_used_at.in_time_zone('EST').strftime('%B %-d, %Y %H:%M Eastern Time'),
```

Comment:
We probably want to use US & Canadian eastern time zone instead of the international eastern time zone.